docs/contributing.md

```diff
@@ -72,6 +72,48 @@ Running the tests
 docker run --rm -ti -v $(pwd):/usr/src/app opsdroid/opsdroid:myfeature tox
 ```
 
+## Automatic Linting with Black
+
+Opsdroid is running black to deal with linting issues. Black will be triggered when you `git commit` your changes.
```

Comment:
You actually need to set up [precommit](https://pre-commit.com/) in your development environment for this to happen. Might be worth documenting that.

The linting CI also now includes black checks.